### PR TITLE
Tweaking defaults for better perfs on higher load

### PR DESCRIPTION
**Pull request by jotak**
these seem to be better defaults to me (empirically tested)
sounds ok to you?

### CODE REVIEW

**Comment by jotak** [comment]
with this kind of config (and also some tweaks on the CR side) I can safely run hey-ho like this: `./hey-ho.sh -z 10s -r 5 -q 2000`
which means 2,5M qps cluster-wide, which I run on a small 3wk, 2 masters cluster  (but I didn't check if these 2,5M QPS were effectively met)

File: examples/kafka/topic.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ metadata:
   labels:
     strimzi.io/cluster: "kafka-cluster"
 spec:
-  partitions: 3
+  partitions: 8
```
Comment (mariomac) [inline examples/kafka/topic.yaml:8]: I'd set something bigger, as long as a single FLP reader can handle multiple partitions with relatively small cost. E.g. 24 or 32.

Comment (jotak) [reply examples/kafka/topic.yaml]: done

File: hack/loki-distributed.sh [hunk 2]
```diff
@@ -44,6 +44,10 @@ kubectl apply -n $NAMESPACE -f examples/distributed-loki/3-services/
 echo ""
 echo "Deployment complete"
 echo ""
+echo "Configure FlowCollector Loki with:"
+echo "    url: 'http://loki-distributed-distributor.netobserv.svc.cluster.local:3100/'"
+echo "    querierUrl: 'http://loki-distributed-query-frontend.netobserv.svc.cluster.local:3100/'"
```
Comment (OlivierCazade) [inline hack/loki-distributed.sh:49]: I think, the namespace here should be network-observability here

Comment (jotak) [reply hack/loki-distributed.sh:49]: the default namespace changed: https://github.com/netobserv/documents/pull/26/files#diff-d828b4887a8f430cebe46d5cc46bed8578de7639942a7e66bb6bfa5ad2b55220

Comment (jotak) [reply hack/loki-distributed.sh:49]: well, actually it should be `${NAMESPACE}`

Comment (jotak) [reply hack/loki-distributed.sh]: (done)

**Review by mariomac — APPROVED** [review]

File: hack/loki-distributed.sh (hunk 2 above)
Comment (OlivierCazade) [reply hack/loki-distributed.sh]: Missed this changed, sorry.

**Review by OlivierCazade — APPROVED** [review]

**Pull request merged**